### PR TITLE
Vet clinic database: add 'join table' for visits

### PR DESCRIPTION
**Pull request by normainobary2021**
In this PR, I did the following:
- Create a table named vets and two join table specializations and visits
- Insert new data
- Write queries to answer the following:
-- Who was the last animal seen by William Tatcher?
-- How many different animals did Stephanie Mendez see?
-- List all vets and their specialties, including vets with no specialties.
-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
-- What animal has the most visits to vets?
-- Who was Maisy Smith's first visit?
-- Details for most recent visit: animal information, vet information, and date of visit.
-- How many visits were with a vet that did not specialize in that animal's species?
-- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

I have also attached the following query screenshots:

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -50,3 +50,43 @@ SET owner_id =
...
+       ('Stephanie Mendez', 64, '1981-05-04'),
+       ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert the following data into the specializations table
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon')),
+       ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon')),
+       ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon')),
+       ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```
Comment (Davidosky007) [inline data.sql]: * I get this error when I tried to run these queries Kindly fix them. HINT ( you can try adding the `id` number for each vet and species  example: `william tatcher: 1`, `pokemon: 1`  ) you can get this `ids` from the qureies used to insert data into the species table and vets table. Do this fix for all your queries that invloves `vet_id` and `species_id`
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @normainobary2021 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Well written PR description 👍 
* Good loking readme file 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you @Davidosky007, I made the changes.

**Review by Whoistolu — APPROVED** [review]
### Hi @normainobary2021 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you so much @Whoistolu for the great and positive review

**Pull request merged**